### PR TITLE
Add descending SRS order (Fixes #281)

### LINKED ISSUE #281: Feature Request: Descending SRS order
Like (closed) #24, but add descending SRS order.

### PR DESCRIPTION
**Pull request by turtlemaster19**
Fix #281 by adding descending SRS order

### CODE REVIEW

**Comment by turtlemaster19** [comment]
Oops! I still need to fix `Main.storyboard`. 🤦‍♂

File: ios/Settings.h [hunk 1]
```diff
@@ -29,11 +29,12 @@
 
 typedef NS_ENUM(NSUInteger, ReviewOrder) {
   ReviewOrder_Random = 1,
-  ReviewOrder_BySRSStage = 2,
-  ReviewOrder_CurrentLevelFirst = 3,
```
Comment (davidsansome) [inline ios/Settings.h]: The existing numbers need to stay the same, so when people upgrade from an old version their settings stay the same.

Can you add ReviewOrder_DescendingSRSStage to the end as number 7?
It's okay to rename ReviewOrder_BySRSStage to ReviewOrder_AscendingSRSStage though - the names don't matter.

Comment (turtlemaster19) [reply ios/Settings.h]: Done, thanks.

**Comment by davidsansome** [comment]
Thanks!

**Pull request merged**